lib/chromedriver.js

```diff
@@ -216,22 +216,33 @@ class Chromedriver extends events.EventEmitter {
 
   async killAll () {
     let cmd;
+    let cmdAdb;
     if (system.isWindows()) {
       // js hint cannot handle backticks, even escaped, within template literals
       cmd = "FOR /F \"usebackq tokens=5\" %a in (`netstat -nao ^| " +
             "findstr /R /C:\"" + this.proxyPort + " \"`) do (" +
             "FOR /F \"usebackq\" %b in (`TASKLIST /FI \"PID eq %a\" ^| " +
             "findstr /I chromedriver.exe`) do (IF NOT %b==\"\" TASKKILL " +
             "/F /PID %a))";
+      cmdAdb = "FOR /F \"usebackq tokens=2\" %a in (`adb forward --list ^| findstr /R /C:\"webview\"`) do ( adb forward --remove %a )";
     } else {
       cmd = `pkill -15 -f "${this.chromedriver}.*--port=${this.proxyPort}"`;
+      cmdAdb = "for a in `adb forward --list | grep \"webview\" | awk '{print $2}'` ; do  adb forward --remove $a ; done";
     }
-    log.info(`Killing any old chromedrivers, running: ${cmd}`);
+    const cp_exec = B.promisify(cp.exec);
+    log.debug(`Killing any old chromedrivers, running: ${cmd}`);
     try {
-      await (B.promisify(cp.exec))(cmd);
-      log.info("Successfully cleaned up old chromedrivers");
+      await cp_exec(cmd);
```

Comment:
this will work, but it will not use any of the machinery we have around use of `adb` via the `appium-adb` project. are we sure it will be compatible with all of the assurances we have from using that project? (for example, finding adb at various paths).

i guess one option could be to pass an instance of `ADB` into the constructor, which could then be used in this method.